firefox-ios/firefox-ios-tests/Tests/XCUITests/TopTabsTest.swift

```diff
@@ -361,7 +361,7 @@ class TopTabsTest: BaseTestCase {
         navigator.nowAt(NewTabScreen)
         waitForTabsButton()
         navigator.goto(TabTray)
-        app.swipeDown()
+        mozWaitForElementToExist(tabsTrayCell.firstMatch)
```

Comment:
@dragosb01 What do you think about `mozWait` as a way to ensure that we can `swipeUp()`in the next step?

Reply:
I am fine with it.